### PR TITLE
Update FAudio, changed version number to 19.03

### PR DESCRIPTION
**Pull request by Zemogiter**
Ready for review.

### CODE REVIEW

File: Engines/Wine/Verbs/FAudio/script.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ var verbImplementation = {
         var wine = new Wine();
         wine.prefix(container);
         var wizard = SetupWizard(InstallationType.VERBS, "FAudio", java.util.Optional.empty());
-        var versions = ["19.02", "19.01"];
+        var versions = ["19.03", "19.02", "19.01"];
         var selectedVersion = wizard.menu(tr("Please select the version."), versions, "19.02");
```
Comment (ImperatorS79) [inline Engines/Wine/Verbs/FAudio/script.js]: I think you can default to latest.

Comment (Zemogiter) [reply Engines/Wine/Verbs/FAudio/script.js]: I've tried to do it like in `DXVK` verb but https://raw.githubusercontent.com/Kron4ek/FAudio-Builds/master/RELEASE gives me 404 not found error.

Comment (ImperatorS79) [reply Engines/Wine/Verbs/FAudio/script.js]: I just meant to change 19.02 to 19.03.

Comment (Zemogiter) [reply Engines/Wine/Verbs/FAudio/script.js]: Ah ok. Done.

**Pull request merged**